Hearthstone Deck Tracker/Importing/CardImageImporter.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Hearthstone_Deck_Tracker.Hearthstone.Entities;
+using Hearthstone_Deck_Tracker.Utility.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hearthstone_Deck_Tracker.Importing
+{
+	public static class CardImageImporter
+	{
+		const string HearthstoneArtUrl = "https://art.hearthstonejson.com/v1/256x";
+		private static string StoreImagesPath = Path.Combine(Config.AppDataPath, "\\Images");
+
+		private static List<string> _succesfullyDownloadedImages = new List<string>();
+		private static Dictionary<string, Task> _inProcessDownloads = new Dictionary<string, Task>();
+
+		static CardImageImporter()
+		{
+			if(!Directory.Exists(StoreImagesPath))
+				Directory.CreateDirectory(StoreImagesPath);
```

Comment:
`CreateDirectory` can throw various exceptions and needs to go into a try-catch: https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.createdirectory?view=netcore-3.1